flexvalue/db.py

```diff
@@ -161,11 +161,7 @@ def get_filtered_acc_gas(database_version, start_year, end_year):
     -------
     pd.DataFrame
     """
-    columns = [
-        "year",
-        "month",
-        *ACC_COMPONENTS_GAS,
-    ]
+    columns = ["year", "month", *ACC_COMPONENTS_GAS]
```

Comment:
interesting that the formatter changed this back

Reply:
Yeah, not sure what to make of that!